### DISCUSSION TITLE
23.11.29 데일리 스크럼

### DISCUSSION THREAD

**Discussion by Jeonwoojae** [petit-a-petit/MIML#79] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 서비스 테스트 작성
- 내일은 어떤 일을 하나요?
    - 테스트 촘촘히 짜기

**Reply by minju412** [reply]
기능별로 문서화하여 테스트 케이스를 촘촘히 짤 수 있을 것 같다! (PR에 작성)

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - PR 피드백 받은 내용 수정
- 내일은 어떤 일을 하나요?
  - 플레이리스트 음악 저장 기능 구현
  - 플레이리스트 알림 기능 연동
  - 친구 추가 알림 기능 연동
